test/hotspot/jtreg/runtime/CompressedOops/CompressedCPUSpecificClassSpaceReservation.java

```diff
@@ -120,8 +120,10 @@ private static void do_test(boolean CDS) throws IOException {
     }
 
     public static void main(String[] args) throws Exception {
-        System.out.println("Test with CDS");
-        do_test(true);
+        // SapMachine 2025-06-11: -Xshare:on does not work with -XX:-UseCompactObjectHeaders
+        // because of the reduced number of jsa cds archives
```

Comment:
Maybe better:
-Xshare:on can only be used for a configuration for which we have a jsa cds archive. We only have one for UseCompressedOops + UseCompactObjectHeaders.

Reply:
I think we better document more in the issue or at some place where we write about the incompatibility.

Reply:
I think "-XX:-UseCompactObjectHeader" is confusing because "@requires vm.flagless".

Reply:
I think you can omit the last sentence of my proposal:
```
// SapMachine 2025-06-11: reduce number of cds/jsa archives
// -Xshare:on can only be used for a configuration for which we have a jsa cds archive.
```
would be fine.